covid19_sfbayarea/data/san_mateo/cases_by_ethnicity.py

```diff
@@ -0,0 +1,13 @@
+from typing import Dict, List
+from .power_bi_querier import PowerBiQuerier
+
+class CasesByEthnicity(PowerBiQuerier):
+    def __init__(self) -> None:
+        self.source = 'c'
+        self.name = 'cases_by_race'
+        self.property = 'race_cat'
+        super().__init__()
+
+    def _parse_data(self, response_json: Dict[str, List]) -> Dict[str, int]: # type: ignore
```

Comment:
This is because I am using inheritance poorly and returning different types with subclasses. Any refactor ideas?